src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -588,26 +588,19 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environmental and recording system's
-noise.
-In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
```

Comment:
A single empty room recording should exist for any given session. Not more.

Reply:
I disagree with this.  Are you saying that if I collect multiple noise recordings, I can't have them all in my BIDS dataset?  See also my comment below about the AssociatedNoiseRecording field.